command-palette/ui-tests/README.md

```diff
@@ -0,0 +1,114 @@
+# Test
```

Comment:
Maybe we should also diff these files on CI to make sure they stay in sync?

- `ui-tests/README.md`
- `ui-tests/playwright.config.ts`

Reply:
Done thanks for the suggestion
> Note I added test for the server-extension too.